kahuna/public/js/leases/leases.css

```diff
@@ -133,8 +138,13 @@ gr-leases .gu-date-range__display {
 gr-leases.leases-flex {
   display: flex;
   flex-direction: row;
+  width: 100%;
 }
 
+.result-editor__field-container__leases {
```

Comment:
minor: could we [reuse existing](https://github.com/guardian/grid/blob/master/kahuna/public/stylesheets/main.css#L327) classes?